earthfile2llb/earthfile2llb.go

```diff
@@ -77,10 +77,13 @@ type ConvertOpt struct {
 	// AllowPrivileged is used to allow (or prevent) any "RUN --privileged" or RUNs under a LOCALLY target to be executed,
 	// when set to false, it prevents other referenced remote targets from requesting elevated privileges
 	AllowPrivileged bool
-	// DoSaves is used to control when SAVE ARTIFACT AS LOCAL calls will actually output the artifacts locally
-	// this is to differentiate between calling a target that saves an artifact directly vs using a FROM which indirectly
-	// calls a target which saves an artifact as a side effect.
+	// DoSaves controls when SAVE ARTIFACT AS LOCAL, and SAVE IMAGE (to the local docker instance) calls are executed
+	// When a SAVE IMAGE --push is encountered, the image may still pushed to the remote registry (as long as DoPushes=true), but is
```

Comment:
(Sorry, this was my error :-) )

```suggestion
	// When a SAVE IMAGE --push is encountered, the image may still be pushed to the remote registry (as long as DoPushes=true), but is
```